config/resources/concept-scheme.lisp

```diff
@@ -0,0 +1,26 @@
+(define-resource concept-scheme ()
+  :class (s-prefix "skos:ConceptScheme")
+  :properties `(
+    (:uuid :string ,(s-prefix "core:uuid"))
+    (:label :string ,(s-prefix "skos:prefLabel"))
+    )
```

Comment:
What predicates do we want to have here? I see that there is also a `note`, `modified` predicate?